### ISSUE TITLE
[ 13:04:40 ] (*) Solved Captcha | P1_eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.hKdwYXNza2V5xQbV Error: Account authorization key not found in the system or has incorrect format

### ISSUE THREAD

**Issue by Shadowsmashsui** [H4cK3dR4Du/Discord-Member-Booster#78]
[ 13:04:40 ] (*) Solved Captcha | P1_eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.hKdwYXNza2V5xQbV
Error: Account authorization key not found in the system or has incorrect format this error is by me all times

**Comment by TherealONE12** [comment]
same at me!

**Comment by NxyyLOL** [comment]
get a captcha key by signing up and depositing funds @ capsolver.com or capmonster.cloud

**Comment by TherealONE12** [comment]
> get a captcha key by signing up and depositing funds @ capsolver.com or capmonster.cloud

I have put my Capsolver key in it. But i also get tha error. I only did where it said "inout capsha key and there i did, without adding anything else"

**Comment by NxyyLOL** [comment]
so uhhh did u enable proxy scraper + add me on disc nxy_done

**Comment by TherealONE12** [comment]
Yeah I have it enabled but I only habe a vapsolver key, not 2capsha/capmonster

**Comment by NxyyLOL** [comment]
then add me on discord nxy_done

Nxyy

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by CaptainExorY** [comment]
I have the same issue.. have the key and proxy scrape enabled
